Sources/EpoxyBars/BarInstaller/BarWrapperView.swift

```diff
@@ -156,29 +156,29 @@ public final class BarWrapperView: UIView {
     }
   }
 
-  private func setModel(_ model: InternalBarCoordinating?, animated: Bool) {
+  private func _setModel(_ originalModel: BarModeling?, animated: Bool) {
     let oldValue = _model
-
-    guard let originalModel = model else {
+    
+    guard let underlyingBarModel = originalModel?.internalBarModel else {
       view?.removeFromSuperview()
       view = nil
       _coordinator = nil
       return
     }
-
-    let coordinator = self.coordinator(for: originalModel)
-
-    guard let model = originalModel.barModel(for: coordinator).internalBarModel as? InternalBarModeling else {
+    
```

Comment:
I think the lint check is failing due to all of these lines with extra whitespace